fg2: <main> function()
  block 0: [entry]
    node 0: constant[undefined,v1] (test-resources/src/flowgraphbuilder/flowgraph_builder0045.js)
    node 1: function-decl[function f(x),-] (test-resources/src/flowgraphbuilder/flowgraph_builder0045.js:1:1)
    ->[block 1]
  block 1:
    node 2: constant["tajs-host-env:string-replace-model.js",v2] (host-environment-sources-loader)
    node 3: constant[true,v3] (host-environment-sources-loader)
    ->[block 2]
  block 2:
    node 4: call[TAJS_load,v2,v3,v4] (host-environment-sources-loader)
    ->[block 3]
  block 3:
    node 5: call[-,v4,-] (host-environment-sources-loader)
    ->[block 4]
  block 4:
    node 6: constant["tajs-host-env:error-captureStackTrace-model.js",v5] (host-environment-sources-loader)
    node 7: constant[true,v6] (host-environment-sources-loader)
    ->[block 5]
  block 5:
    node 8: call[TAJS_load,v5,v6,v7] (host-environment-sources-loader)
    ->[block 6]
  block 6:
    node 9: call[-,v7,-] (host-environment-sources-loader)
    ->[block 7]
  block 7: [exit-ordinary]
    node 10: return[v1] (test-resources/src/flowgraphbuilder/flowgraph_builder0045.js)
    ->[]
  block 8: [exit-exceptional]
    node 11: exceptional-return (test-resources/src/flowgraphbuilder/flowgraph_builder0045.js)
    ->[]
function f(x)
  block 9: [entry]
    node 12: constant[undefined,v1] (test-resources/src/flowgraphbuilder/flowgraph_builder0045.js:1:1)
    ->[block 10]
  block 10:
    node 13: read-variable['x',v1,-] (test-resources/src/flowgraphbuilder/flowgraph_builder0045.js:1:23)
    ->[block 11]
  block 11: [exit-ordinary]
    node 14: return[v1] (test-resources/src/flowgraphbuilder/flowgraph_builder0045.js:1:1)
    ->[]
  block 12: [exit-exceptional]
    node 15: exceptional-return (test-resources/src/flowgraphbuilder/flowgraph_builder0045.js:1:1)
    ->[]

test-resources/src/flowgraphbuilder/flowgraph_builder0045.js: [definite] Exceptional program exit is unreachable
test-resources/src/flowgraphbuilder/flowgraph_builder0045.js:1:1: [definite] Unreachable function f
Soundness testing succeeded for 2 checks (with 0 expected failures)
